src/views/List.jsx

```diff
@@ -1,11 +1,19 @@
 import { ListItem } from '../components';
 
-export function List({ data }) {
+export function List({ data, setSearchQuery }) {
 	return (
 		<>
-			<p>
-				Hello from the <code>/list</code> page!
-			</p>
+			<div>
+				<label htmlFor="filter">Filter Items</label>
+				<br />
+				<input
```

Comment:
Since your input is on the list you'd want to have the state and the filtering in the list component as well...I don't think it makes any sense for the input to be on the List component and the state and the filtering on the parent.